spec/models/callback_modifier_spec.rb

```diff
@@ -29,7 +29,7 @@
           it 'should default to after destroy' do
```

Comment:
And I pushed the version comparison fix I made in #813 as a new amended commit: cd4c667f5dfd306748b87c355f39d00bbb53ea86